client/app.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+
+class App extends React.Component{
+    constructor(props){
+    super(props)
+    }
+
+
+
+    render() {
+        return (
+            <div>
+            <div>
```

Comment:
Indentation looks a little wonky here.  Are you using a linter? I see you downloaded one, but I think we need to make sure we apply it (or at least try to) because I think we are going to get judged based on code cleanliness as well.